Content.Server/GameTicking/GameTicker.cs

```diff
@@ -135,6 +135,8 @@ public override void Initialize()
             _configurationManager.RegisterCVar("game.defaultpreset", "Suspicion", CVar.ARCHIVE);
             _configurationManager.RegisterCVar("game.fallbackpreset", "Sandbox", CVar.ARCHIVE);
 
+            _configurationManager.RegisterCVar("game.disablewin", false, CVar.CHEAT);
```

Comment:
This CVar should be renamed to `game.enablewin`, to keep it consistent with the rest:
- discord.enabled
- outline.enabled
- parallax.enabled
- game.lobbyenabled
- log.enabled
- loki.enabled
- metric.enabled
- status.enabled

```suggestion
            _configurationManager.RegisterCVar("game.enablewin", false, CVar.CHEAT);
```